txpool/txpool_test.go

```diff
@@ -855,7 +856,139 @@ func TestSizeLimit(t *testing.T) {
 			err = pool.addImpl(tt.input.origin, genTx(t, &tt.input))
 			assert.Equal(t, tt.err, err)
 			assert.Equal(t, tt.len, pool.Length())
-			assert.Equal(t, tt.slots, pool.slots)
+			assert.Equal(t, tt.slots, pool.slotsOccupied())
+		})
+	}
+}
+
+func TestGaugeCheck(t *testing.T) {
+	type AddTx struct {
+		nonce    uint64
+		gasPrice *big.Int
+		slotSize uint64
+		origin   TxOrigin
+	}
+
+	genTx := func(t *testing.T, arg AddTx) *types.Transaction {
```

Comment:
If this is a helper function mark it as such. I don't feel that we should define helper functions inside the tests themselves though.

Reply:
[Fixed(https://github.com/0xPolygon/polygon-sdk/pull/213/commits/ce80cf9d857d332b6c1258016cbccee67526cfbd)